codex-ui/src/vue/components/alert/Alert.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <transition name="fadeInBottom">
+    <div
+      v-show="message"
+      class="alert"
+      :class="`alert--${alertType}`"
+    >
+      <Icon name="Check" />
```

Comment:
icon should be optional and configurable

Reply:
added